From ddfe45f33ca156e2cb40fc80bd1146618825c62f Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Thu, 25 Oct 2012 17:04:50 +0800
Subject: [PATCH 360/449] ENGR00231176 V4L2 output unit test: Add tiled pixel
 format support

V4L2 output unit test: Add IPU_PIX_FMT_TILED_NV12 and
IPU_PIX_FMT_TILED_NV12F format support.

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index a5bcf45..398284d 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -463,6 +463,8 @@ int process_cmdline(int argc, char **argv)
 			     (g_in_fmt != V4L2_PIX_FMT_YUV422P) &&
 			     (g_in_fmt != IPU_PIX_FMT_YUV444P) &&
                              (g_in_fmt != V4L2_PIX_FMT_YUV420) &&
+                             (g_in_fmt != IPU_PIX_FMT_TILED_NV12) &&
+                             (g_in_fmt != IPU_PIX_FMT_TILED_NV12F) &&
                              (g_in_fmt != V4L2_PIX_FMT_NV12) )
                         {
                                 return -1;
-- 
1.8.0

